### PR TITLE
objects.txt: Add various recent OIDs including those registered for CMPv3

### PR DESCRIPTION
**Pull request by DDvO**
This PR adds ~92 mostly PKIX-related~ OIDs registered at IANA ~, including those~ for the upcoming CMP-Updates RFC (see https://datatracker.ietf.org/doc/draft-ietf-lamps-cmp-updates/), which defines extensions to CMPv2 and the new CMP version 3.
This is ~rather bulky, but~ auto-generated stuff, in `crypto/objects/` and `obj_mac.h`.

### CODE REVIEW

**Comment by DDvO** [comment]
The **out-of-source-and-install (macos-latest)** CI failure must be unrelated.

File: crypto/objects/objects.txt [hunk 1]
```diff
@@ -493,6 +493,90 @@ id-pkix-mod 13		: id-mod-timestamp-protocol
...
+id-pkix-mod 91		: id-mod-ip-addr-and-as-ident-2v2
+id-pkix-mod 92		: id-mod-lamps-eai-addresses-2016
+id-pkix-mod 93		: id-mod-safecurves-pkix
+id-pkix-mod 94		: id-mod-pkix1-shakes-2019
+id-pkix-mod 95		: id-mod-delegated-credential-extn
+id-pkix-mod 96		: id-mod-MASAURLExtn2016
+id-pkix-mod 97		: id-mod-anima-acpnodename-2020
+id-pkix-mod 98		: id-mod-est-2019
+id-pkix-mod 99		: id-mod-cmp2021-88
+id-pkix-mod 100		: id-mod-cmp2021-02
```
Comment (paulidale) [inline crypto/objects/objects.txt:579]: Indentation??

**Review by paulidale — COMMENTED** [review]
Are all of these necessary?  Are they useful now?

**Comment by levitte** [comment]
Why is this separated from #15778?

File: crypto/objects/objects.txt (hunk 1 above)
Comment (DDvO) [reply crypto/objects/objects.txt:579]: This apparent extra indentation is an artifact of the `git diff` output due to the added `+`, which caused the tab following the `100` to jump in the diff output. In the editor it is fine.
BTW, in the automatic insertion I used the same two tabs per line that were present for the existing lines before.

**Comment by DDvO** [comment]
> Why is this separated from #15778?

For two reasons:
* In order to make #15778 less bulky
* Since I included also the in-between OID decls not related to CMPv3, which may be useful for other purposes.

**Comment by DDvO** [comment]
> Are all of these necessary? Are they useful now?

At the moment they are not used, yet the CMP-related ones are going to be used for CMPv3.
I included the other ones just for sake of completeness, but if you prefer I can drop the `id-pkix-mod` entries 17..49 and 51..98.
Though I presume implementors of other recent IETF RFCs in the PKIX realm etc. will be grateful to have them.

**Comment by DDvO** [comment]
> > Why is this separated from #15778?
> 
> For two reasons:
> 
>     * In order to make #15778 less bulky
> 
>     * Since I included also the in-between OID decls not related to CMPv3, which may be useful for other purposes.

A further reason is that all these decls are already official, not just in draft state.
So IMHO it would make sense to have them in 3.0

**Comment by paulidale** [comment]
There is a space overhead for including OIDs.  If we have ones that aren't used, it costs everyone.

I know we've plenty of examples already but that's not a reason to perpetrate more.  Adding a new OID at run time is one call...

**Comment by DDvO** [comment]
Rebased in order to get rid of the unrelated CI failure for MacOS.

**Comment by DDvO** [comment]
> There is a space overhead for including OIDs. If we have ones that aren't used, it costs everyone.
> 
> I know we've plenty of examples already but that's not a reason to perpetrate more. Adding a new OID at run time is one call...

I see, so I'm removing those new ones not related to CMP.

BTW, `crypto/objects/objects.pl` does not cope well with deleting entries from `crypto/objects/objects.txt`.

**Comment by DDvO** [comment]
BTW, `crypto/objects/objects.pl` does not cope well with deleting entries 
from `crypto/objects/objects.txt`. For each of them, it retains a line like this:
```
    { NULL, NULL, NID_undef },
```

**Comment by t8m** [comment]
I think this should be moved to Post 3.0

**Comment by DDvO** [comment]
> I think this should be moved to Post 3.0

All right. Then I can also move the remaining changes back to #15778.
Done now, so closing this PR.

**Pull request closed without merging**